rust/src/blocks/cb_csv.rs

```diff
@@ -0,0 +1,248 @@
+use crate::block::Block;
+use crate::core::do_blocking;
+use crate::core::log;
+use crate::core::registerBlock;
+use crate::types::common_type;
+use crate::types::ClonedVar;
+use crate::types::Context;
+use crate::types::OptionalString;
+use crate::types::ParamVar;
+use crate::types::Parameters;
+use crate::types::Seq;
+use crate::types::Table;
+use crate::types::Type;
+use crate::types::SEQ_OF_STRINGS_TYPES;
+use crate::types::STRING_TYPES;
+use crate::CString;
+use crate::Types;
+use crate::Var;
+use core::convert::TryFrom;
+use core::convert::TryInto;
+use csv::Reader;
+use csv::ReaderBuilder;
+use csv::WriterBuilder;
+
+lazy_static! {
+  static ref PARAMETERS: Parameters = vec![
+    (
+      cstr!("NoHeader"),
+      cbccstr!("Whether the block should parse the first row as data, instead of header."),
+      vec![common_type::bool]
+    )
+      .into(),
+    (
+      cstr!("Separator"),
+      cbccstr!("The character to use as fields separator."),
+      vec![common_type::string]
+    )
+      .into()
+  ];
+  static ref STR_HELP: OptionalString =
+    OptionalString(cbccstr!("A multiline string in CSV format to parse."));
+  static ref SEQ_HELP: OptionalString = OptionalString(cbccstr!(
+    "A sequence of rows, each row being a sequence of strings derived parsing the input string."
```

Comment:
The reference to input will also look weird when that help is applied to `inputHelp()`.

Suggestion: "A sequence of rows, with each row being a sequence of strings."